eng/pipelines/windows.yml

```diff
@@ -115,14 +115,27 @@ stages:
         buildExtraArguments: /p:RuntimeOS=win10
 
         variables:
-          - uapNetfxQueues: Windows.10.Amd64.ClientRS5.Open
-          - windowsArmQueue: Windows.10.Arm64.Open
+          - ${{ if eq(parameters.isInternalBuild, 'false') }}:
+            - nanoQueues: "`(Windows.Nano.1809.Amd64.Open`)windows.10.amd64.serverrs5.open@mcr.microsoft.com/dotnet-buildtools/prereqs:nanoserver-1809-helix-amd64-08e8e40-20200107182504"
```

Comment:
In the linux files the parens were escaped with \, here it's `.  Is that expected?

Reply:
This was there before and a merge from 3.1 removed it - not sure why they were inconsistent. Both are valid escapes afaik.